doc/dev-reference/getting_started.rst

```diff
@@ -168,10 +171,10 @@ Common options for CMake include:
 |                        | Valid options are Debug, Release, RelWithDebInfo,  |
 |                        | and MinSizeRel. Default is RelWithDebInfo.         |
 +------------------------+----------------------------------------------------+
-| BUILD_SHARED_LIBS      | Build Hyperscan as a shared library instead of     |
+| BUILD_SHARED_LIBS      | Build Vectorscan as a shared library instead of    |
 |                        | the default static library.                        |
 +------------------------+----------------------------------------------------+
-| BUILD_STATIC_AND_SHARED| Build both static and shared Hyperscan libs.       |
+| BUILD_STATIC_AND_SHARED| Build both static and shared Vectorscan libs.      |
```

Comment:
could you please amend this according to the options in the README/wiki ?

Reply:
This should be done